commons/monitor/questionnaire/questionnaire_application_usage_log.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "namespace": "org.radarcns.monitor.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireApplicationUsageLog",
+  "doc": "Schema of reporting events when opening the app, clicking notifications, and starting and finish questionnaires.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "eventType", "type": {
+      "name": "UsageEventType",
+      "type": "enum",
+      "doc": "Android activity usage event type.",
+      "symbols": ["NOTIFICATION_OPEN", "APP_OPEN", "NOTIFICATION_FOREGROUND", "QUESTIONNAIRE_STARTED", "QUESTIONNAIRE_FINISHED", "QUESTIONNAIRE_CLOSED", "OTHER", "UNKNOWN"]
+      }, "doc": "Whether the app was opened directly or by clicking a notification, a questionnaire was started, completed, closed, etc.", "default": "UNKNOWN" }
+  ]
```

Comment:
perhaps add a questionnaire type field as well which will be null by default.

Reply:
Perhaps after the `time` or `eventType` field also add a `name` field to distinguish questionnaire type from its name.

Reply:
The end of the line of questionnaireType contains a comma too much.

Reply:
Sorry what do you mean by to distinguish the questionnaire type from its name? I think questionnaireType is just the name?